configure.py

```diff
@@ -138,6 +138,7 @@ def try_compile_and_link(compiler, source='', flags=[], verbose=False):
             if verbose:
                 ret = subprocess.run([compiler, '-x', 'c++', '-o', ofile, sfile.name] + args.user_cflags.split() + flags,
                                     capture_output=True)
+                print(ret.stdout.decode('utf-8'))
```

Comment:
I would add printing the command line attempted too, so the user doesn't have to guess what's happening.

Reply:
And maybe the source (if the `source` parameter was passed to try_compile, and descriptive string too.

Reply:
Ah, I see this isn't going into a `./configure.py --verbose` thing, but rather something just for a few call sites.

So it makes sense to print the command line only if we had a failure.

Reply:
yup, this parameter is internal and is only used in the error path